spotify-artist-viewer/Models/SAArtistTableViewController.h

```diff
@@ -7,14 +7,13 @@
 //
 
 #import <UIKit/UIKit.h>
+#import "SARequestManager.h"
 
 @interface SAArtistTableViewController : UITableViewController
-//<UISearchBarDelegate, UISearchDisplayDelegate>
 @property (strong, nonatomic) NSMutableArray *artistArray;
```

Comment:
Unless its really necessary don't use a data type in a property name. Just call this artists.